tests/test_mig_install_generateconfs.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+#
+# --- BEGIN_HEADER ---
+#
+# test_mig_install_geneateconfs - unit test of generateconfs
+# Copyright (C) 2003-2024  The MiG Project by the Science HPC Center at UCPH
+#
+# This file is part of MiG.
+#
+# MiG is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# MiG is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301,
+# USA.
+#
+# --- END_HEADER ---
+#
+
+"""Unit tests for the generateconfs utility"""
+
+from tests.support import MigTestCase, testmain
+from mig.install.generateconfs import _PARAMETERS
```

Comment:
`make PY=2 test` is unhappy about this import:
```
0| ../migrid-pr83 > make PY=2 test
validating container.. rebuilt for changes
running with MIG_ENV='local' under python 2

.......E......sss.....ssss.
Running unit test on shared core functions
Testing shared base functions
Lookup client_id_dir for /C=DK/ST=NA/L=NA/O=NBI/OU=NA/CN=Jonas Bardino/emailAddress=bardino@nbi.ku.dk
Client dir: +C=DK+ST=NA+L=NA+O=NBI+OU=NA+CN=Jonas_Bardino+emailAddress=bardino@nbi.ku.dk
Lookup client_dir_id for +C=DK+ST=NA+L=NA+O=NBI+OU=NA+CN=Jonas_Bardino+emailAddress=bardino@nbi.ku.dk
Client id: /C=DK/ST=NA/L=NA/O=NBI/OU=NA/CN=Jonas Bardino/emailAddress=bardino@nbi.ku.dk
Lookup get_short_id for /C=DK/ST=NA/L=NA/O=NBI/OU=NA/CN=Jonas Bardino/emailAddress=bardino@nbi.ku.dk
Client short id: bardino@nbi.ku.dk
Visible vs invisible path tests
Check script restrictions:
Check brief format list limit
Completed shared base functions tests
Done running unit test on shared core functions
..E....
======================================================================
ERROR: test_mig_install_generateconfs (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_mig_install_generateconfs
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/local/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/usr/src/app/tests/test_mig_install_generateconfs.py", line 31, in <module>
    from mig.install.generateconfs import _PARAMETERS
ImportError: No module named install.generateconfs


======================================================================
ERROR: test_mig_shared_install (unittest.loader.ModuleImportFailure)
----------------------------------------------------------------------
ImportError: Failed to import test module: test_mig_shared_install
Traceback (most recent call last):
  File "/usr/local/lib/python2.7/unittest/loader.py", line 254, in _find_tests
    module = self._get_module_from_name(name)
  File "/usr/local/lib/python2.7/unittest/loader.py", line 232, in _get_module_from_name
    __import__(name)
  File "/usr/src/app/tests/test_mig_shared_install.py", line 44, in <module>
    from mig.install.generateconfs import _GENERATE_CONFS_PARAMETERS
ImportError: No module named install.generateconfs
```

Reply:
Yep - it's rather a weird behavioural difference between the versions it seems: the easiest fix appears to be dropping an __init__.py in mig/install - not my preference, given that folder is mostly templates and I feel the python files should ideally be elsewhere, but for the sake of pragmatism I'm going to add it so things work and we can move forward. Incidentally doing so raises another Python 2 gotcha (which I happen to be addressing as we speak) but otherwise what's here will not change further.

Reply:
Fixed as of the most recent push.